examples/bookstore/booklistproxymodel.cpp

```diff
@@ -76,7 +77,7 @@ QModelIndex BookListProxyModel::index(int row, int column, const QModelIndex &pa
   if (parent.isValid())
     return QModelIndex();
 
-  return createIndex(row, 0, 0);
```

Comment:
The third argument here is to select the createIndex overload that puts an "id" into the model index.
http://doc.qt.io/qt-5/qabstractitemmodel.html#createIndex-1

Probably ok as the "id" value is like 0 by default, but maybe this can be fixed to be sure?